src/types.ts

```diff
@@ -86,6 +98,7 @@ export interface ITokenData {
   uid: string;
   name: string;
   symbol: string;
+  version: TokenVersion;
```

Comment:
Maybe the version should be optional for backwards compatibility.
In the event of connecting with an old fullnode/service that does not take token version into account, having it required could break something, what do you think?